src/Cqm/CqmClient.php

```diff
@@ -25,20 +25,23 @@ class CqmClient extends HttpClient
     protected function getCommand(): string
     {
         $port = $this->port;
-        $node = "node"; //$GLOBALS['node_binary'];
+        $node = $GLOBALS['node_binary'];
         $cmd = $this->servicePath;
 
         if (IS_WINDOWS) {
             $cmd = "start /B $node $cmd";
             return $cmd;
         } else {
+            $command = $node;
             $system = new System();
```

Comment:
Need to test if the global is populated. Most admins are not going to know where the node binary is located much less know to populate the global. Installing node via NVS will not set an OS global path to binary like install via PPA or MSI on windows. This is generally how node is installed and should not require us setting a path to node binary. Even for MacOS. So at minimum we should set the node binary if missing in code e.g:

```
if (IS_WINDOWS) {
            if (empty($node)) {
                $node = 'node';
            }
            $cmd = "start /B $node $cmd";
            return $cmd;
        } else {
            if (empty($node)) {
                $node = 'nodejs';
            } 
... 
```